### ISSUE TITLE
[Suggestion/Proposal] Vancouver Open source Summit NA Informal Meetup

### ISSUE THREAD

**Issue by berryq460** [cncf/tag-env-sustainability#91]
As many of you may know, this group is actively working to promote sustainable practices in the open source community, and I believe that hosting a meetup is a great way to contribute to this effort.

I will be at VC and would love to join forces with other members to host a meetup

I have proposed the idea already in more detail here (https://github.com/cncf/tag-env-sustainability/issues/43#issuecomment-1412433095), and I am looking for cohosts who are passionate about sustainability and open source to help make this event a success.

If you are interested in cohosting this meetup, please reach out to me directly or leave a comment. Together, we can make a positive impact on the open source community and the planet!

**Comment by leonardpahlke** [comment]
cool! ref: https://github.com/cncf/tag-env-sustainability/issues/88

**Comment by berryq460** [comment]
yes I saw that for the blog, I added this one for the meetup. Thanks!

**Comment by leonardpahlke** [comment]
I think @nikimanoledaki is around! Unfortunately, I will not be attending :(

**Comment by berryq460** [comment]
Got it, did you enjoy Amsterdam though! Hope it was fruitful and your talks/meetups went well!

**Comment by leonardpahlke** [comment]
Kubecon was great 🤩. The meetups, even if it's just a few new people, help a lot! I would love for you to organize a meetup at OSS NA! @berryq460 if you want to be in the driver's seat here, that would be greatly appreciated!

**Comment by berryq460** [comment]
Delighted to hear that! By the way, How did you all advertise the meetup? At your talk? Slack group? In person ? Will definitely be down to do this with some support

**Comment by leonardpahlke** [comment]
idea: If there is a talk at the conference that is of particular interest to the TAG, you could contact the speaker and arrange to have an informal meeting with anyone interested. If the speaker announces this at the end of the talk, many of the attendees might come to the meeting :) (or go with any other idea 😄)

**Comment by leonardpahlke** [comment]
For Kubecon we got support from the CNCF to organize a room and such (we will do so also for Kubecon NA). So the meetup appeared on sched. For this one we could send out tweets & slack.

**Comment by berryq460** [comment]
Love that idea! There are quite a few talks on the sustainability track. I will go for a familiar Red Hat contact but ask if @nikimanoledaki if you can assist me on reaching out. I can make a flyer as well to share in group chats

**Comment by berryq460** [comment]
![CNCF TAG - Earth Hour Meetup Flyer](https://user-images.githubusercontent.com/24463725/234424183-32dd8006-13bd-49d3-9520-7a154bfdd925.png)

**Comment by rootfs** [comment]
I'll be there on Thu.

**Comment by catblade** [comment]
Can we add this to the poster (handles QR code to website with our official logo):

![image](https://user-images.githubusercontent.com/1184214/234871058-1264c6e7-7192-49bc-8a4c-4ecb4fec5b28.png)

**Comment by berryq460** [comment]
You got it. This Qr code is EPIC

**Comment by berryq460** [comment]
2nd iteration:  ![CNCF TAG - Earth Hour Meetup Flyer (1)](https://user-images.githubusercontent.com/24463725/234983737-3d2427d9-3dc7-49fb-8de4-43d3e4cd9125.png)

**Comment by berryq460** [comment]
Second iteration ![CNCF TAG - Earth Hour Meetup Flyer (2)](https://user-images.githubusercontent.com/24463725/234984176-92f32830-254f-4397-90fa-35e482176e10.png)

**Comment by berryq460** [comment]
Another option ![CNCF TAG - Earth Hour Meetup Flyer (4)](https://user-images.githubusercontent.com/24463725/234986113-2c6c7707-502a-435a-acb5-711e3948097b.png)

**Comment by berryq460** [comment]
New flyer coming up with the location @nikimanoledaki graciously proposed! Do you mind reminding me the room number? Thanks for your support on this. FYI: Side note for anyone reading related to this meetup: https://github.com/cncf/tag-env-sustainability/issues/61#issuecomment-1534941240

**Comment by nikimanoledaki** [comment]
Thank you for driving this, @berryq460! 😄

Looking at the [SustainabilityCon track](https://ossna2023.sched.com/overview/type/SustainabilityCon?iframe=no), it is happening mostly in Room 112, so I would suggest the following for the location: `Meeting Point: Outside Room 112 (Level 1), Canada Place` - what do you think?

With regard to the time, there is a gap from 12:45pm that we can take advantage of 👀

**Comment by berryq460** [comment]
Perfect! Sounds like a plan 😄 Thanks for helping coordinate! I will add
those details to the flyer and then we can post it on social media !

At the same time we can aim to reach out to 1-2 speakers at the
Sustainability panel and see who is down to promote and join us! What do
you think? Looking forward to meeting!

**Comment by berryq460** [comment]
Can someone post this on Twitter please ![CNCF TAG - Earth Hour Meetup Flyer (5)](https://user-images.githubusercontent.com/24463725/236559502-b736e341-08a6-4478-9fbe-04dea36a82ef.png)

**Comment by leonardpahlke** [comment]
Sure, give me minute

**Comment by berryq460** [comment]
WE ARE LIVE !![ PLEASE Repost/ Reshare for visibility](https://twitter.com/CNCFEnvTAG/status/1654591315013779458)

**Comment by berryq460** [comment]
Related to this, I will be at the conference all week and would like new folk to see this at the meetup today, can you please Edit directly on my behalf adding the images in the following sections? PR Done https://github.com/berryq460/tag-env-sustainability/pull/3 

 Thanks so much! : Here is the original doc:  https://docs.google.com/document/d/117Mt6sDInSxdsVA-vBI9Cwls67FvQbU2KR1WqqKY0U8/edit?usp=sharing

**Comment by nikimanoledaki** [comment]
Hey @leonardpahlke! I was mostly offline last week but here are some notes about the conference:

## cdCon + GitOpsCon
### Talk on energy + carbon monitoring
Started off on Monday with a talk by @Al-HusseinHameedJasim and me where we presented the energy benchmark test results. 🎉 Will share the recording in the TAG's Slack channel as soon as it is uploaded, which I was told should be in a week or so. Once it's uploaded, it would be great to add it as an agenda point to discuss as a group at an upcoming meeting of the TAG. What do you think?

## OS Summit
### Informal meetup
@berryq460 and I met on Wednesday. It was nice to meet in person! We didn't manage to gather a group but, personally, there was a lot of awareness work done through individual interactions throughout the week, so we reached people that way.

### SustainabilityCon
I gathered some notes in [this thread](https://twitter.com/NikiMnldk/status/1656698493220831235) and [posted on Slack](https://cloud-native.slack.com/archives/C03F270PDU6/p1683825281976779).

TL;DR: There are multiple groups in the Linux Foundation working on Scope 1/2/3 monitoring, reduction, and how to do this in a formal way that follows any established standards.

### Sustainability Talks
Huge shoutout to @wangchen615 for delivering the following two very interesting talks on cloud-native sustainability at OSS, with the second one featuring @rootfs:
- [Optimizing Full-Stack Sustainability in a Real World Data Center: Practices and Lessons Learned - Chen Wang, Hua Ye & Fan Jing Meng, IBM](https://ossna2023.sched.com/#)
- [5 Steps to Deploy Cloud Native Sustainable Foundation AI Models - Chen Wang, IBM & Huamin Chen, Red Hat](https://ossna2023.sched.com/#)

As soon as the recordings are uploaded, I will make sure to share them with you all on the TAG's Slack channel as well. There is a lot to learn from these talks and I can't wait to rewatch and study the details more carefully 👏 

### TAG interest
There was a lot of interest in the ENV TAG in general. There are 6 people in particular that I would like to follow up with to share more info about the TAG. @leonardpahlke was the intro to the TAG recorded or could it be a recurring event for newcomers?

**Comment by nikimanoledaki** [comment]
@leonardpahlke sounds good, will open a PR in the next days (ETA Monday) :)

**Comment by berryq460** [comment]
Hey so I found out the day of that the room number was incorrectly chosen on the flyer and that the time  slot was also at the Diversity luncheon so instead of the meetup I pivoted to just organic conversations on the go with members at the luncheon which was perfect because I met folk were they were. It was a great time, great food and conversations both sustainability and non sustainability related. Given my personal experiences to date with the group I didn't pitch this group but gave it a mention to folk at the conference.  Cheers.
